### PR TITLE
explorer: 11939 normalize token values largest table

### PR DESCRIPTION
**Pull request by oJshua**
#### Problem
The token values in the largest account table for an SPL Token do not use uniform decimals and therefore are difficult to compare despite being ordered. % of Total Supply values show 0.000% when holdings are really small relative to total supply.

#### Summary of Changes
Uses common fixed point for balance (picks largest mantissa). Adds tilde for approximation.

Fixes https://github.com/solana-labs/solana/issues/11939

### CODE REVIEW

**Comment by jstarry** [comment]
Nice! I think that it could still be clearer. What do you think about making the values monospace?

Before
Screen Shot 2020-09-04 at 3 12 21 PM

After
Screen Shot 2020-09-04 at 3 12 07 PM

**Comment by jstarry** [comment]
I think adding commas for the numbers would help too

**Comment by oJshua** [comment]
@jstarry Oh nice, I was thinking it was already monospaced for some reason. That'll help. Commas will go a long way too.

File: explorer/src/components/account/TokenLargestAccountsCard.tsx [hunk 1]
```diff
@@ -102,8 +122,21 @@ const renderAccountRow = (
...
-      <td className="text-right">{percent}</td>
+      <td className="text-right text-monospace">
+        {fixedLocaleNumber(account.uiAmount, balanceFixedPoint)}
+      </td>
+      <td className="text-right text-monospace">{percent}</td>
     </tr>
   );
 };
+
+function fixedLocaleNumber(value: number, fixedPoint: number) {
```
Comment (jstarry) [inline explorer/src/components/account/TokenLargestAccountsCard.tsx:133]: Nice solution! You can also use this API: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat

**Review by jstarry — APPROVED** [review]

**Pull request merged**